manifests/certificate/x509.pp

```diff
@@ -115,7 +131,7 @@
   $_csr = pick($csr, "${_csr_dir}/${name}.csr")
   $_key = pick($key, "${_key_dir}/${name}.key")
 
-  if !empty($altnames) {
+  if !empty($altnames+$extkeyusage) {
```

Comment:
Ah right, I hadn't seen that this already existed. Although I think `$req_ext = !empty($altnames+$extkeyusage)` would do exactly the same.